vllm/model_executor/models/__init__.py

```diff
@@ -95,6 +101,12 @@ def load_model_cls(model_arch: str) -> Optional[Type[nn.Module]]:
     def get_supported_archs() -> List[str]:
         return list(_MODELS.keys())
 
+    @staticmethod
+    def register_out_of_tree_model(model_arch: str,
```

Comment:
nit: What about `register_model` or `register_custom_model`?

Reply:
fixed in [c51050f](https://github.com/vllm-project/vllm/pull/3871/commits/c51050f9a4ea3705ab72bcaf72c06eae2d955d6d)